knora/dsplib/utils/excel_to_json_properties.py

```diff
@@ -44,7 +45,8 @@ def properties_excel2json(excelfile: str, outfile: str) -> list[dict[str, Any]]:
     # load file
     wb = load_workbook(filename=excelfile, read_only=True)
     sheet = wb.worksheets[0]
-    props = [row_to_prop(row) for row in sheet.iter_rows(min_row=2, values_only=True, max_col=13)]
+    props = [row_to_prop(row) for row in sheet.iter_rows(min_row=2, values_only=True, max_col=13)
+             if any(row) and any([re.search(r'[A-Za-z]+', elem) for elem in row if isinstance(elem, str)])]
```

Comment:
Without this check, openpyxl goes through the entire Excel sheet, which produces a lot of rows that only contain `None` or cells with whitespaces in it. The regex is to check if there are actually words in a cell, not only whitespaces.

Reply:
what about numbers etc.? I think there should be a method on strings to check if it's whitespace. This might prove more reliable in some edge cases (and may even be faster)

Reply:
It's not necessary to check edge cases, because as soon as only one letter is present in one cell, the entire row is processed. Actually, this is rather too permissive than too strict.